controllers/servers/csi/csi_controller_server.py

```diff
@@ -595,7 +595,30 @@ def CreateVolumeGroup(self, request, context):
 
     @csi_method(error_response_type=csi_pb2.DeleteVolumeGroupResponse, lock_request_attribute="volume_group_id")
     def DeleteVolumeGroup(self, request, _):
-        raise NotImplementedError()
+        secrets = request.secrets
+        utils.validate_delete_volume_group_request(request)
+
+        try:
+            volume_group_id_info = utils.get_volume_group_id_info(request.volume_group_id)
+        except ObjectIdError as ex:
+            logger.warning("volume group id is invalid. error : {}".format(ex))
+            return csi_pb2.DeleteVolumeGroupResponse()
+
+        array_type = volume_group_id_info.array_type
+        volume_group_id = volume_group_id_info.ids.uid
+        array_connection_info = utils.get_array_connection_info_from_secrets(secrets)
+
+        with get_agent(array_connection_info, array_type).get_mediator() as array_mediator:
+            logger.debug(array_mediator)
```

Comment:
do we want to expose it without any additional content?

Reply:
sure, we are doing it for most methods

Reply:
so why do we do it in some places, and in some places not? 
and it not better to just put it in the `get_mediator` function?

Reply:
Im not sure why, and maybe.. would you like me to open a ticket for it?

Reply:
it will reduce all the places with `logger.debug(array_mediator)`, yes open it thanks